src/layer/arm/shufflechannel_arm.cpp

```diff
@@ -378,6 +393,21 @@ int ShuffleChannel_arm::forward_bf16s_fp16s(const Mat& bottom_blob, Mat& top_blo
                     ptr1 += 8;
                     outptr0 += 8;
                 }
+
+                for (int i = 0; i < 8; i++)
+                {
+                    if (i % 2)
+                    {
+                        *outptr0 = *ptr1;
+                        ptr1 += 1;
+                    }
+                    else
+                    {
+                        *outptr0 = *ptr0;
+                        ptr0 += 1;
+                    }
+                    outptr0 += 1;
+                }
```

Comment:
I don't think there is a risk of out of range reading in some of the arm pack8 codes.